hack/lib/cleanup.sh

```diff
@@ -86,18 +101,33 @@ readonly -f os::cleanup::dump_container_logs
 # Returns:
 #  None
 function os::cleanup::internal::list_k8s_containers() {
```

Comment:
We have some downstream users of this code, could remove this func but didn't want to be backwards incompatible in case someone was using it directly.